src/lib/VirtualScroller.js

```diff
@@ -372,6 +374,53 @@ class VirtualScroller {
         newListEl.style.height = `${this.totalItems * (this.itemHeight + this.margin) + this.margin}px`;
         return newListEl;
     }
+
+    /**
+     * Scrolls the provided row index into view.
+     * @param {number} rowIndex - the index of the row in the overall list
+     * @return {void}
+     */
+    scrollIntoView(rowIndex) {
+        if (!this.scrollingEl || rowIndex < 0 || rowIndex >= this.totalItems) {
+            return;
+        }
+
+        // See if the list item indexed by `rowIndex` is already present
+        const foundItem = Array.prototype.slice.call(this.listEl.children).find((listItem) => {
+            const { bpVsRowIndex } = listItem.dataset;
+            const parsedRowIndex = parseInt(bpVsRowIndex, 10);
+            return parsedRowIndex === rowIndex;
+        });
+
+        if (foundItem) {
+            // If it is already present and visible, do nothing, but if not visible
+            // then scroll it into view
+            if (!this.isVisible(foundItem)) {
+                foundItem.scrollIntoView();
+            }
+        } else {
+            // If it is not present, then adjust the scrollTop so that the list item
+            // will get rendered.
+            const topPosition = (this.itemHeight + this.margin) * rowIndex;
+            this.scrollingEl.scrollTop = topPosition;
+        }
+    }
+
+    /**
+     * Checks to see whether the provided list item element is currently visible
+     * @param {HTMLElement} listItemEl - the list item elment
+     * @return {boolean} Returns true if the list item is visible, false otherwise
+     */
+    isVisible(listItemEl) {
+        if (!this.scrollingEl || !listItemEl) {
+            return false;
+        }
+
+        const { scrollTop } = this.scrollingEl;
+        const { offsetTop } = listItemEl;
+
+        return scrollTop <= offsetTop && offsetTop <= scrollTop + this.containerHeight;
```

Comment:
Can you put a () around `scrollTop + this.containerHeight`? Reading this I saw
`scrollTop <= offsetTop && offsetTop <= scrollTop` first 😮